Formula/symfony.rb

```diff
@@ -0,0 +1,18 @@
+class Symfony < Formula
+  desc "Installs the Symfony framework for PHP"
+  homepage "https://symfony.com/"
+  url "https://github.com/symfony/cli/releases/download/v4.7.0/symfony_darwin_amd64.gz"
```

Comment:
Precompiled binaries aren't allowed. Any homebrew formula will have to build from source. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
The Symfony CLI tool is not open source. The binary is the only distribution package available.

Reply:
Then it's not a good fit for homebrew, that's also in the link I send